src/stream-tcp.c

```diff
@@ -4653,6 +4655,10 @@ int StreamTcpPacket (ThreadVars *tv, Packet *p, StreamTcpThread *stt,
             (PKT_IS_TOCLIENT(p) && (ssn->server.flags & STREAMTCP_STREAM_FLAG_NOREASSEMBLY)))
         {
             p->flags |= PKT_STREAM_NOPCAPLOG;
+            /* we can call bypass callback, if enabled */
+            if (StreamTcpBypassEnabled() && PacketHasBypassCallback(p)) {
```

Comment:
the comment in suricata.yaml mentions only stream.depth, so it will need to be updated as well